test/stripe/account_link_test.rb

```diff
@@ -7,8 +7,8 @@ class AccountLinkTest < Test::Unit::TestCase
     should "be creatable" do
       link = Stripe::AccountLink.create(
         account: "acct_123",
-        failure_url: "https://stripe.com/failure",
-        success_url: "https://stripe.com/success",
+        refresh_url: "https://stripe.com/refresh",
+        return_url: "https://stripe.com/return",
         type: "custom_account_verification"
```

Comment:
Can you change this to `account_onboarding` and pin to stripe-mock 0.94.0 directly? The latest version I released has that other change and so at least it minimizes issues for someone who installed the latest again

Reply:
Sure thing!